jsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "allowSyntheticDefaultImports": false,
```

Comment:
question, why are we setting `allowSyntheticDefaultImports` to false ??

Reply:
Left that in by accident, removed now.